arangodb-net-standard/Transport/Http/HttpApiTransport.cs

```diff
@@ -104,6 +142,27 @@ private static void ApplyHeaders(WebHeaderCollection webHeaderCollection, HttpHe
             return transport;
         }
 
+        /// <summary>
+        /// Get an instance of <see cref="HttpApiTransport"/> that uses
+        /// JWT-Token authentication to connect to the _system database.
+        /// </summary>
+        /// <param name="hostUri"></param>
+        /// <param name="jwtToken"></param>
+        /// <param name="contentType">Content type to use in requests.
+        /// Used to set Content-Type and Accept HTTP headers.
+        /// Uses JSON content type by default.</param>
+        /// <param name="encodedCA">Base64 encoded CA certificate</param>
+        /// <returns></returns>
+        public static HttpApiTransport UsingJwtAuth(
```

Comment:
Do we really need to duplicate each of the auth methods? Why not keep a single method and make the `dbName` parameter default to `"_system"`? Similar to what you did for `UsingNoAuth`.

I think using a default parameter will be simpler and more obvious for callers, rather than having to choose between two overloads.

Reply:
It was easy to specify a default value in `UsingNoAuth` because the parameter `dbName` is located just before the first optional parameter (`contentType`). To pass a default value and make it optional in `UsingBasicAuth` and `UsingJwtAuth` would mean that I would have to change the position of the `dbName` parameter to after all required parameters (as required by C#). This will break existing solutions that are using the driver. That's the reason why I overloaded the methods.

Reply:
@DiscoPYF , please let me know if you can think of a another method to solve this. Thanks.

Reply:
Oh I see, I overlooked that. Then it should be fine, thanks 👍